app/views/layouts/_flash.html.erb

```diff
@@ -8,7 +8,7 @@
         <%= h(message) %>
       </div>
       <div class="d-flex pl-1">
-        <a class="close-icon icon icon-sm pr-1" alt="Close" href="#">&nbsp;<span class="sr-only">Close</span></a>
+        <a class="close-icon icon icon-sm pr-1" alt="<%= l10n("close") %>" href="#">&nbsp;<span class="sr-only"><%= l10n("close") %></span></a>
```

Comment:
Not a showstopper, but maybe we could add `mr-2` to the span? Otherwise PR looks good!